src/book/02-system/03-tools/06-natnet.mdx

```diff
@@ -0,0 +1,104 @@
+
+8. The side panel will now switch to the ground plane tab. Find the Ground Plane L Frame (It is black with 3 MoCap dots on it and two bubble levels), and place in on the ground at the center circle (normally with the Z axis pointing towards the goals with the brick wall behind it. Don't bother trying to level out the L frame as it seems to be a hopeless task). Back on the computer, click the "Set Ground Plane" button. This will produce a pop-up window prompting you to save your calibration. Click save. Congratulations, your calibration is complete.
+
+
+</details>
+
+1. Follow the [Getting Started instructions](/guides/main/getting-started).
+
+2. Start the Motive software, it doesn't matter if motive is running locally or on another machine. Ensure that the [Data Streaming option](https://docs.optitrack.com/motive/data-streaming) in Motive is enabled. This will start sending packets to the Motive defined default multicast address on the local network.
+
+3. Once you have the NUbots codebase running on your OS of choice and Motive running and Streaming packets to the network, NatNet can be run in one of two ways:
+
+<details>
+<summary>On the Robot</summary>
+
+This is the default method for running NatNet. If you wish to output motive packets directly to file for debugging purposes, change the dump_packets option in NatNet.yaml to true. 
+
+1. Ensure that the NatNet role has been installed on the robot.
+
+2. [SSH into the robot](guides/main/getting-started#running-on-a-real-robot).
+
+3. `cd config`
+
+4. `nano DataLogging.yaml`
+
+5. Scroll down to message.input.MotionCapture and set to true. Additionally, set to true any of the sub messages that you want to use such as ".RigidBody". If you want "output.CompressedImage" or "input.sensors" logged, you can set them to true as well. Don't log "input.Image" as this will consume huge amounts of storage.
+
+6. To exit nano, press Ctrl+X, then 'y', then hit enter.
+
+7. Run Natnet using `./b run natnet`
+
+8. After your program has finished, check that a new nbs file has been created in "log/<your_program's_name>/". The filename should be the date and timestamp when the file was run.
+
+9. Exit out of the ssh of the robot.
+
+10. We now need to copy the nbs file off the robot. Use "scp nubots@10.1.1.<robot_id#>:/home/nubots/log/<your_program_name>/<name_of_nbs_file>    ./log/"
+
+11. You now need to extract the information you want from the nbs file. We have a few nbs decoders that have already been written for specific purposes. If one of those does not suit your needs, you will need to either modify an existing nbs extraction file, or write your own. The location of these nbs files is "nubots/tools/nbs_tools/". For example purposes, let's assume you want to extract images and Motion Capture data. To run this program, make sure you are in the nubots home directory, then you would use: "./b nbs extract_images_MoCap ./log/<file_name>  --output ./log/<your_extraction_folder>"
```

Comment:
This nbs tool doesn't exist? also mixture of camel case and snake case :P 
I would recommend to extract images using `./b nbs extract_images <path>` and get the numerical data via json extraction `./b nbs json <path>`.